test/runnable/betterCsharedctor.d

```diff
@@ -0,0 +1,21 @@
+/* REQUIRED_ARGS: -betterC
+ * PERMUTE_ARGS:
+ */
```

Comment:
Can `TEST_OUTPUT` be used here? Currently nothing tests that the ctors/dtors are actually executed.